spark-tpcds-benchmark-runner/src/main/java/com/palantir/spark/benchmark/config/HadoopConfiguration.java

```diff
@@ -74,17 +92,10 @@ default Configuration toHadoopConf() {
                         filesystems().values().stream().flatMap(fsConf -> fsConf.toHadoopConf().entrySet().stream()))
                 .collectToMap()
                 .forEach(hadoopConf::set);
-        hadoopConf.set(CommonConfigurationKeys.FS_DEFAULT_NAME_KEY, getFilesystemBaseUriOrThrow(defaultFilesystem()));
+        hadoopConf.set(CommonConfigurationKeys.FS_DEFAULT_NAME_KEY, defaultFsBaseUriString());
         return hadoopConf;
     }
 
-    default String getFilesystemBaseUriOrThrow(String filesystemName) {
```

Comment:
nit: I think factoring this into a function was useful/clean? so would rather not revert that bit

Reply:
yep got lost in the revert, added it back